src/board-item/styles.scss

```diff
@@ -7,6 +8,10 @@
 /* TODO: use container API instead of styles override */
 .container-override.active {
   box-shadow: cs.$shadow-container-active;
+
+  :global([data-awsui-focus-visible]) & {
```

Comment:
Can we keep using `:focus-visible` here? Board components do not set this attribute directly, so it relies on side effects from the main components and we may forget to change it when switching the main repo to native

Reply:
The problem is that the container doesn't have focus, the handler button does.

I would need to use `:has`, which isn't widely supported yet: `&:has(:focus-visible) {`

Do you have another idea?

Reply:
I think, difficulties to implement can be a data point to push back this design change. I do not remember any requests for this change, so this happening is a little surprising for me, tbh.

Let's discuss more in DM